### DISCUSSION TITLE
Is it possible to combine text prompts? I.e. to detect multiple types of objects in one forward pass.

### DISCUSSION THREAD

**Discussion by dempsey-ryan** [opengeos/segment-geospatial#220] [Q&A] [answered]
e.g., detecting both trees and buildings in a satellite image. I tried inputting both `LangSam.predict(text_prompt="tree, building")` and `LangSam.predict(text_prompt="building, tree")` and they yield fairly differing results. I'm wondering if there is a method to detect multiple object prompts without an inherent hierarchy in the prompts due to the order of the text.

**Comment by giswqs** [comment, marked as answer]
Multiple types are not supported yet. I don't have a timeline when it will be implemented. Contributions are welcome.
